docs/joining/application.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Applying as OpenRail project
+weight: 100
+---
+
+# Applying to become an OpenRail project
+
+The OpenRail Association provides hosting to open source projects within the railway sector. Its goal is to facilitate collaboration across organizations so that the huge challenges of the sector can be addressed in a joint effort, effectively and efficiently. The OpenRail Association is open to all participants of the sector, railway companies, vendors, and research institutes, which commit to working on open source software in an open model.
+
+Hosting an open source project means that the OpenRail Association provides the neutral ground where a diverse set of participants can meet under clear conditions and collaborate openly. The OpenRail Association provides infrastructure (mainly through its [organization on GitHub](https://github.com/openrailassociation)) and support for running projects according to proven open source practices. It also provides legitimacy and visibility for open source projects, and a strong international network of like-minded people from the railway sector.
+
+Development of the open source software in the projects is done by the project participants. This will be members by the OpenRail Association in many cases, but this is not a prerequisite. All projects follow an open governance model, which means that decision are taken transparently and participation in projects is based on merits. The OpenRail Association makes sure that these principles are followed for its projects, but leaves projects a lot of autonomy to organize themselves in the way which is most suitable for the project. Control about the content of the project is fully with the projects. The OpenRail Foundation does not provide funding for development.
```

Comment:
```suggestion
Development of the open source software in the projects is done by the project participants. This will be members of the OpenRail Association in many cases, but this is not a prerequisite. All projects follow an open governance model, which means that decision are taken transparently and participation in projects is based on merits. The OpenRail Association makes sure that these principles are followed for its projects, but leaves projects a lot of autonomy to organize themselves in the way which is most suitable for the project. Control about the content of the project is fully with the projects. The OpenRail Foundation does not provide funding for development.
```